acceptance/repository-cloudant/docker.setup.js

```diff
@@ -3,53 +3,27 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+// This script is for creating database for couchdb3 instance that created by
+// `sh setup.sh`. The setup is separated to two files is because this step
+// is hard to achieve by curl. See README for database configuration.
+
 'use strict';
 
-const _ = require('lodash');
 const async = require('async');
-const docker = new require('dockerode')();
 const http = require('http');
-const ms = require('ms');
-const fs = require('fs');
-
-/**
- * This script creates a new couchDB3 container locally. And it also creates the admin for
- * the container. Type in bash command:
- * ```bash
- * CLOUDANT_PASSWORD=myadmin CLOUDANT_PASSWORD=mypass CLOUDANT_DATABASE=mydb node docker.setup.sh
- * ```
- * to customize the names.
- */
+const chalk = require('chalk');
 
 process.env.CLOUDANT_DATABASE = process.env.CLOUDANT_DATABASE || 'testdb';
 process.env.CLOUDANT_PASSWORD = process.env.CLOUDANT_PASSWORD || 'pass';
 process.env.CLOUDANT_USERNAME = process.env.CLOUDANT_USERNAME || 'admin';
 
-// these are placeholders. They get set dynamically based on what IP and port
-// get assigned by docker.
-process.env.CLOUDANT_PORT = 'TBD';
-process.env.CLOUDANT_HOST = 'TBD';
+process.env.CLOUDANT_PORT = process.env.CLOUDANT_PORT || 5984;
+process.env.CLOUDANT_HOST = process.env.CLOUDANT_HOST || 'localhost';
 process.env.CLOUDANT_URL = 'TBD';
 
-const CONNECT_RETRIES = 30;
-const CONNECT_DELAY = ms('5s');
-
-let containerToDelete = null;
-
+// create database and log the status if success
 async.waterfall(
-  [
```

Comment:
I wonder if we can rewrite the script with `async-await`. Usage of `async` module with callbacks is hard to read :-).

Reply:
I am not sure if my changes in [commit](https://github.com/strongloop/loopback-next/pull/4776/commits/2306e0e44ff6a51f7cbaf5c6afed1c9db76a0984) is correct async-await way to run the script :sweat_smile: I only changed the waterfall part.

Reply:
Looks better.